docs/more-pushmsg.md

```diff
@@ -83,3 +83,36 @@ switch ($code) {
         echo "<strong>Hmm Something Went Wrong or HTTP Status Code is Missing</strong>";
 }
 ```
+
+### Node.js - Using Axios
+
+```javascript
+////////////////////////////////////////
+// Send Push Message to Gotify Server //
+////////////////////////////////////////
+
+var url = "http://localhost:8008/message?token=<apptoken>";
+var bodyFormData = {
+  title: "Hello from Javascript",
+  message: "Test Push Service from Node.js",
+  priority: 5
+};
+axios({
```

Comment:
Should you `require()` axios first?

Reply:
Nope, actually I use Gotify Notification system on My Node script - https://github.com/mskian/ssl-expiry-reminder/blob/90ec22a15f8fb56c8d18832c7ab8177261f18f61/check.js#L43

Reply:
In your script you do:
```
const axios = require('axios');
```

Reply:
yes, so Shall I add step Fully?